src/schema/v2/user.ts

```diff
@@ -239,6 +239,44 @@ export const UserType = new GraphQLObjectType<any, ResolverContext>({
         type: GraphQLBoolean,
         resolve: async ({ data_transfer_opt_out }) => data_transfer_opt_out,
       },
+      dislikedArtworksConnection: {
```

Comment:
**question [non-blocking]:** unused connection

I don't believe we need to expose this connection because I don't think we have a use-case for showing a user their disliked artworks. But lmk if I'm missing something!

*format via [Conventional Comments](https://conventionalcomments.org/)*

Reply:
no your right, we only need to collect and store dislikes for now!

Reply:
Yup, this kind of thing should live on `me`! All the goodies are there